packages/embark/src/cmd/cmd_controller.js

```diff
@@ -160,6 +160,9 @@ class EmbarkController {
 
         engine.registerModuleGroup("coreComponents");
         engine.registerModuleGroup("stackComponents");
+        engine.registerModuleGroup("namesystem");
+
+        engine.registerModule('embark-embarkjs', {plugins: engine.plugins});
```

Comment:
Crucially, I believe this needs to be a stack component as it has a lot of `setCommandHandler`s.

Reply:
agree, it's sort of why it was put in the top next to the other stack components, I guess I need to add it to the stackComponents group together with namesystem